ref/general/what-is-open-liberty.adoc

```diff
@@ -16,26 +16,35 @@
+With Open Liberty, you can add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile. This modular structure simplifies microservice development, so you run Just enough Application Server to support the features that your application needs. Furthermore, with Open Liberty zero migration architecture, you can upgrade to the latest version with minimal impact to your current applications and configurations. Open Liberty is compatible with the Jakarta EE Full Platform and Web Profile specifications and with MicroProfile. For the latest updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
 
 Open Liberty offers developers the following benefits:
+
 * A lightweight edition that corresponds to the Java EE Web Profile specification.
 * Excellent development and production runtime environments for web applications.
 * A smaller footprint for faster download and startup, giving more development time and faster time to deployment.
 * Ease of packaging applications for deployment, including configuration.
 * The ability to run applications that are written for Open Liberty on WebSphere Application Server traditional.
 * The ability to extend the Open Liberty capabilities by adding custom features that use a product extension System Programming Interface (SPI).
 
+== Zero migration
+
+With the Open Liberty zero migration policy, you focus on writing your applications, not the APIs changing under you. Zero-migration architecture means that you can use existing, unmodified configuration and application files with an updated version of Open Liberty without unwanted or unexpected change in behavior. Open Liberty offers full compatibility between product versions and pluggable features so that your existing APIs and behaviors are supported in each new product version.
+
+Open Liberty never modifies user configuration files, which are fully compatible between versions. You can use a single version of your configuration files across multiple versions. Files that you created for a previous version of Open Liberty can be used with a later version. Files that you create for later versions can be used with previous versions. As a result, if all configured features are installed, you can use a single set of configuration files across multiple versions without modifications. Any configuration settings that do not apply to a particular version of Open Liberty are ignored.
```

Comment:
Change,
"Open Liberty never modifies user configuration files, which are fully compatible between versions."
to,
"Open Liberty doesn't modify user configuration files, which are fully compatible between versions."